fixed travis error by fixing test-stripe.py, added donation_successful page

rohithsivakumar opened the pull request:
-This PR fixes the error that was showing up in Travis, caused by a syntax error in test-stripe.py
-It also adds the donation_successful page that users are lead to after donating with the stripe form
-These changes were tested by running the app in localhost
<img width="1280" alt="screen shot 2017-06-02 at 1 39 28 pm" src="https://cloud.githubusercontent.com/assets/6231630/26738301/997b4764-479b-11e7-8589-f7cf6722c135.png">

rohithsivakumar committed:
Merge remote-tracking branch 'ARM-open/master'

rohithsivakumar committed:
fixed travis error (fixed test-stripe.py), added donation_succesful page

rohithsivakumar committed:
fixed travis error, STRIPE_KEY is now STRIPE_SECRET_KEY in travis.yml

davidawad commented on .travis.yml:
```diff
@@ -14,7 +14,7 @@ install: "pip install -r requirements.txt"
 env:
   global:
-    - STRIPE_KEY="SAMPLESTRIPEKEY"
+    - STRIPE_SECRET_KEY="SAMPLESTRIPEKEY"
```
🔥

davidawad commented on templates/donation_successful.html:
```diff
@@ -1,5 +1,194 @@
+
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+
+	<!-- <link rel="stylesheet" href="../static/bower_components/font-awesome/css/font-awesome.min.css"> Resource style -->
```
o_0 
wait why are we using the font-awesome cdn and not the local one?

rohithsivakumar replied:
I'm not sure, I just copy-pasted the code over from index.html's "head", and that's how it was. Also, there's no static/bower_components directory in the repo

davidawad replied:
*handled offline*

davidawad commented on templates/donation_successful.html:
```diff
@@ -1,5 +1,194 @@
+
+<title>ARM | Archangel Raphael's Mission </title>
+
+<style>
```
use a separate css style sheet please :)

davidawad commented on templates/donation_successful.html:
```diff
@@ -1,5 +1,194 @@
+    ga('create', '{{GA_ID}}', 'auto');
+    ga('send', 'pageview');
+    </script>
+-->
```
Why is google analytics commented out here?

rohithsivakumar replied:
I wasn't sure if you wanted to use the same GA_ID from the index.html file, since this is a different page. So, I just commented it out for now

davidawad replied:
yes, use the same GA_ID it's fine :)

davidawad commented on templates/donation_successful.html:
```diff
@@ -1,5 +1,194 @@
+
+        	<div id="text">
+        		<h1>Thank You!</h1>
+        		<h4>YOUR SUPPORT IS APPRECIATED</h4>
```
lol, was this really the copy dorie used in the pdf?

davidawad commented on templates/donation_successful.html:
```diff
@@ -1,5 +1,194 @@
+            </nav> <!-- .cd-main-nav-wrapper -->
+
+            <a href="#0" class="cd-nav-trigger">Menu<span></span></a>
+        </header>
```
I think we can leverage jinja templating and avoid repeating so much html in multiple places. 

put this into a header.html and use jinja to `include` that file here and on `index.html`.

That way we only have to maintain one copy of how the header looks.

davidawad reviewed (changes requested):
I have a few changes to request but overall good work!

rohithsivakumar committed:
added "header.html" template file, updated bower stuff

These changes were made in response to the comments on the PR. I tested
these changes by running the app on localhost.

davidawad commented on templates/donation_successful.html:
```diff
@@ -1,5 +1,87 @@
+
+        		</ul>
+        		<h6>Wanna know what ARM is up to? Follow us!</h6>
+        		<a href="#" target='_blank'><b class="link-button">Join Us</b></a>
```
broken link? Fill this in with any link you want from the other join us buttons, it just feels funny that this would be broken haha.

rohithsivakumar committed:
updated "join us" link in donation_successful, updated jquery references, updated bower.json with latest version of jquery

rohithsivakumar committed:
font resource files are now accessed over https

rohithsivakumar committed:
font resources back to loading over http (basically undos previous commit)

DavidAwad commented:
Last thing to iron out here is just to see if we can avoid requesting checkout.js multiple times. 
good work so far @rohithsivakumar
